controllers/ssp_controller.go

```diff
@@ -337,6 +337,8 @@ func preUpdateStatus(request *common.Request) error {
 	sspStatus.OperatorVersion = operatorVersion
 	sspStatus.TargetVersion = operatorVersion
 
+	reconcileSSPResourceMessage := "Reconciling SSP resources"
```

Comment:
I don't fully agree. Strings like this can still be (or are) part of the public API and can be something people expect to be the same.
It's not only about code semantics, but consistency and possibility of human error.

Right now if somebody changes the wording, they have to find all occurrences of the string and also hope there is no typo in any of them. Having this cleaned up into either a constant like here or having a function for creating the condition reduces those risks and makes it easier for the future.